src/SarifTests/SarifUtilitiesTests.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.CodeAnalysis.Sarif;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.CodeAnalysis.Sarif.Tests
+{
+    [TestClass()]
+    public class SarifUtilitiesTests
+    {
+        private const string V1_0_0 = "1.0.0";
+        private const string V1_0_0_BETA_5 = "1.0.0-beta.5";
+
+        [TestMethod()]
+        public void ConvertToSchemaUriTest1()
```

Comment:
Try to provide informative test names, so that somebody looking at the list of tests in the Test Explorer window knows what they do. In this case the names "test 1" and "test 2" don't provide any information.

Perhaps `ConvertToSchemaUriV100`, `ConvertToSchemaUriV100Beta5`.